core/src/main/java/bisq/core/offer/OpenOfferManager.java

```diff
@@ -1143,7 +1178,33 @@ private boolean isBsqSwapOfferLackingFunds(OpenOffer openOffer) {
                 openOffer.isBsqSwapOfferHasMissingFunds();
     }
 
+    public boolean isSpam(OpenOffer newOffer) {
```

Comment:
isSpam is a too generic and overloaded term. Better use some more descriptive method name or something with "isValid"